src/Tests/dotnet-new.Tests/Approvals/DotnetNewHelpTests.CanShowAllowScriptsOption.verified.txt

```diff
@@ -5,13 +5,21 @@ Usage:
   dotnet new TestAssets.PostActions.RunScript.Basic [options] [template options]
 
 Options:
-  -n, --name <name>                The name for the output being created. If no name is specified, the name of the output directory is used.
+  -n, --name <name>                The name for the output being created. If no 
```

Comment:
Is this change supposed to change width of output? 
this looks suspicious

Reply:
In this example it's just width, and it is a concern.

Reply:
There's a fairly chunky delta between the SDK this PR updates to and the one that we were previously targeting so I'm not sure what exactly caused this change.

I will say that enforcing a consistent line width on the console output seemed like a sensible bug fix to me.

Where is the repo that these code changes are sourced from?

Reply:
This may be a change in System.CommandLine.  @baronfel